components/blocks/Booking.tsx

```diff
@@ -0,0 +1,104 @@
+        </div>
+      </div>
+    </Link>
+  )
+}
+
+const BookingBlock = ({ data, index }) => {
+  const [meetingPeople, setMeetingPeople] = useState<BookingItem[]>([]);
+
+  useEffect(() => { 
+    const fetchData = async () => {
+      const meetingPeopleData = await fetchMeetingLinks();
+      setMeetingPeople(meetingPeopleData);
+    }
+
+    fetchData();
+  }, []);
+
+  if (!meetingPeople.length) return null;
+
+  return (
+    <div className="flex justify-center">
+      <div className="md:my-18 lg:w-1/2 p-6 sm:pt-1">
+        <div className="container mx-auto px-4">
+          <div className="flex justify-center mb-6">
+            <div className="step-indicator flex items-center">
+              <div className="step flex flex-col items-center relative z-10">
+                <div className="step-icon w-6 h-6 border-2 border-orange-300 flex items-center justify-center rounded-full"></div>
+                <p className="text-center absolute bottom-[-20px] text-xxs md:text-xs whitespace-nowrap">
+                  CHOOSE LOCATION
+                </p>
+              </div>
+              <div className="indicator-line h-0.5 w-20 md:w-40 bg-gray-300"></div>
+              <div className="step flex flex-col items-center relative z-10">
+                <div className="step-icon w-6 h-6 bg-none border-2 border-gray-300 text-white flex items-center justify-center rounded-full"></div>
+                <p className="text-center absolute bottom-[-20px] text-xxs md:text-xs whitespace-nowrap">
+                  CHOOSE TIME
+                </p>
+              </div>
+              <div className="indicator-line h-0.5 w-20 md:w-40 bg-gray-300"></div>
+              <div className="step flex flex-col items-center relative z-10">
+                <div className="step-icon w-6 h-6 bg-none border-2 border-gray-300 flex items-center justify-center rounded-full"></div>
+                <p className="text-center absolute bottom-[-20px] text-xxs md:text-xs whitespace-nowrap">
+                  YOUR INFO
+                </p>
+              </div>
+            </div>
+          </div>
+          <div className="grid gap-3 my-16 px-10 py-12 md:bg-gradient-to-br from-white/25 via-white/50 to-white/75 break-inside-avoid md:shadow-md rounded-lg">
+            <h1 className="w-full text-center inline-block m-0 md:text-4xl font-tuner lg:text-4xl text-2xl whitespace-nowrap lg:leading-tight bg-gradient-to-br from-orange-400 via-orange-500 to-orange-600 bg-clip-text text-transparent">
+              Choose Your Location
+            </h1>
+            <h3 className="w-full text-center mb-6 inline-block m-0 pb-4 text-lg md:whitespace-nowrap lg:leading-tight text-black">
+              Book a meeting with a 🦙TinaCMS expert in your timezone{' '}
+            </h3>
+            {meetingPeople.map((cardItem, idx) => (
+              <BookingCard key={idx} cardItem={cardItem} />
+            ))}
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
This is a lot of divs, can you try clean it up a bit?
Also not sure where this is actually used I don't remember seeing it anywhere

Reply:
Lines 1-86 are the Step Indicator at the top of the page
- each "<div className="step-icon w-6 h-6 border-2 border-orange-300 flex items-center justify-center rounded-full"></div>" is the circle 

- each <div className="indicator-line h-0.5 w-20 md:w-40 bg-gray-300"></div> is the line between each circle

Reply:
✅ though